web/src/components/profile/settings/FreezeAccount.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, useState } from 'react';
+import { Button } from 'web/src/components/ui/Button';
+import { useT } from 'web/src/hooks/useT';
+import { Modal, ModalFooter, ModalHeader } from 'web/src/components/ui/Modal';
+import { Box } from 'web/src/components/ui/Box';
+import { useFreezeProfile } from 'web/src/hooks/mutations/useUpdateProfile';
+
+export const FreezeAccount: FC = () => {
+  const t = useT();
+  const [open, setOpen] = useState(false);
+  const freezeAccount = useFreezeProfile();
+
+  const handleClick = () => setOpen(true);
+  const handleClose = () => setOpen(false);
+  const handleFreeze = () => {
+    freezeAccount();
+    setOpen(false);
+  };
+
+  const strong = (...chunks: string[]) => <strong>{chunks}</strong>;
+
+  return (
+    <>
+      <Button color="secondary" onClick={handleClick}>
+        {t('Freeze Account')}
+      </Button>
+      <Modal size="lg" show={open} onClose={handleClose}>
```

Comment:
```suggestion
      <Modal size="sm" show={open} onClose={handleClose}>
```
Поменяй размер, будет ближе к дизайну